tests/health_checks/NDv4/test-bandwidth.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+
+#Lock the graphics clock.
+sudo nvidia-smi -lgc 1400 > /dev/null
+
+#Count the GPUs.
+ngpus=$(nvidia-smi --query-gpu=name --format=csv,noheader | wc -l)
+
+#Download the host to device and device to host test from superbench v0.5.
+string="https://raw.githubusercontent.com/microsoft/superbenchmark/release/0.5"\
+"/superbench/benchmarks/micro_benchmarks/gpu_copy_performance/gpu_copy.cu"
+wget -q $string
+
+#Compile the gpu-copy benchmark.
+nvcc -I/usr/include/ -L/usr/lib/x86_64-linux-gnu/ -lnuma gpu_copy.cu -o gpu-copy
```

Comment:
We could do that, it would make the script a bit simpler. I just thought it would be good to be transparent about exactly what test is being run. Should we switch?

Reply:
yes, i would vote for including the compiled test in the image.

Reply:
Based on our conversation offline I have made it so that if nvcc is detected the copy_test_file.sh script downloads and compiles the gpu-copy test. Does this work better? 

https://github.com/moes1/azhpc-images/blob/master/common/copy_test_file.sh#L10